src/main/java/uk/gov/pay/api/model/CreateCardPaymentRequest.java

```diff
@@ -165,6 +171,13 @@ public String getReturnUrl() {
         return Optional.ofNullable(internal);
     }
 
+
+    @JsonProperty("set_up_agreement")
+    @Schema(description = "agreement ID", required = false, example = "abcefghjklmnop1234567890", hidden = true)
```

Comment:
Completely trivial but the example here is 24 characters and wouldn't actually pass the validation